dataRetriever/resolvers/topicResolverSender/diffPeerListCreator_test.go

```diff
@@ -118,7 +154,8 @@ func TestDiffPeerListCreator_PeersListNoExcludedTopicSetShouldRetPeersOnMain(t *
 			},
 		},
 		mainTopic,
-		excludedTopic,
+		intraTopic,
+		emptyTopic,
```

Comment:
I would use directly "" and I would remove declaration for emptyTopic as it is used only here or otherwise I would also use it in lines 39 and 54

Reply:
it serves an explanatory purpose. The compiler replace the string at compile time, anyway